charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -267,6 +267,12 @@ spec:
             value: {{ .Values.datadog.prometheusScrape.version | quote }}
           {{- end }}
           {{- end }}
+          - name: DD_CHART_RELEASE_NAME
```

Comment:
nit: `DD_` is used for Agent parameters, meaning that you will have log lines about unknown parameters due to that.
As these are not read from Agent configuration, I'd remove the `DD_`